meilisearch-http/src/routes/stats.rs

```diff
@@ -200,28 +209,29 @@ pub(crate) fn report(pid: Pid) -> SysInfo {
     info.global.used_memory = sys.get_used_memory();
     info.global.total_swap = sys.get_total_swap();
     info.global.used_swap = sys.get_used_swap();
-    info.global.input_data = sys.get_networks().into_iter().map(|(_, n)| n.get_received()).sum::<u64>();
-    info.global.output_data = sys.get_networks().into_iter().map(|(_, n)| n.get_transmitted()).sum::<u64>();
+    info.global.input_data = sys
+        .get_networks()
+        .into_iter()
+        .map(|(_, n)| n.get_received())
+        .sum::<u64>();
+    info.global.output_data = sys
+        .get_networks()
+        .into_iter()
+        .map(|(_, n)| n.get_transmitted())
+        .sum::<u64>();
 
-    if let Some(process) = sys.get_process(pid) {
+    if let Some(process) = sys.get_process(data.server_pid) {
```

Comment:
Did you add a new `server_pid` field to the `Data` struct? Why?

Reply:
Where does this `pid` came from before?

Reply:
Previously the `pid` come from `ctx.state().server_pid`. It's exactly the same here. I haven't modified the Data struct.

Reply:
Previously the pid come from ctx.state().server_pid. It's exactly the same here. I haven't modified the Data struct.